model_config:
  vae_path: model_3
  latent_dim: 3
  TD3_path: 0